src/app/course/course-question-snippet/course-question-snippet.component.html

```diff
@@ -1,4 +1,15 @@
 <ng-container *ngIf="uqjs">
+    <button
+        *ngIf="event.type==='ASSIGNMENT'||event.type==='EXAM'"
```

Comment:
There should be spaces between operators.
```suggestion
        *ngIf="event.type === 'ASSIGNMENT' || event.type === 'EXAM'"
```